CHANGELOG.md

```diff
@@ -3,6 +3,41 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## nf-core/ampliseq version 2.4.2 - 2023-03-01
```

Comment:
```suggestion
## nf-core/ampliseq version 2.5.0 - 2023-03-01
```
After reviewing, this has a ton of added functionality. Which is a good thing, it's just not a patch in my understanding of semantic versioning.

Reply:
Alright, I am fine with naming it 2.5.0, did so in https://github.com/nf-core/ampliseq/pull/550